modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -3031,12 +3031,33 @@ definition {
 	}
 
 	@description = "Verify that a message warning is not displayed when the user adds another transition and do not selects it as default"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test WarningIsNotDisplayedWhenNewTransitionNotDefault {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#WarningIsNotDisplayedWhenNewTransitionNotDefault";
+
+		Workflow.openWorkflowListView();
 
-		// TODO LRQA-73833 WarningIsNotDisplayedWhenNewTransitionNotDefault pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Task",
+			nodePosition = "-20,60");
+
+		ProcessBuilderKaleoDesignerReact.backArrowOnSidebar();
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "task",
+			connectorLabel = "Start-Task Connector",
+			connectorOut = "start",
+			key_direction = "right");
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start-End Connector",
+			connectorOut = "start",
+			key_direction = "right");
 
```

Comment:
You need to add a step to **turn on** the default, and then **turn off** to really see if there is not alert present. 

`ProcessBuilderKaleoDesignerReact.clickDefaultToggle(defaultConnector = "True");
Alert.viewWarningSpecific(warningMessage = "Default Was Changed:Your workflow may have changed because the default was changed.");`

i`ve added the macro  `viewWarningSpecific` to make sure that when the default is turned on, the alert appears, but this is optional, we already have another test `[WarningIsDisplayedWhenNewTransitionDefault]` checking this, later you do the turn on/turn off action, you can use/create the macro `viewWarningSpecificNotPresent` to assert that the alert is not there. Thanks